### DISCUSSION TITLE
［求助贴］急急急急急急急！

### DISCUSSION THREAD

**Discussion by aminoc12138** [share121/inter-knot#1091] [其他]
被一只无敌的蜗牛追杀了怎么办！！在线等！！急！
![1722962223078](https://github.com/user-attachments/assets/50b7d74a-e6c6-4325-a6a6-e76643fc42d5)

**Comment by chenger2024** [comment]
吃点好的吧

**Comment by mr99710** [comment]
建议抓到科研所

**Comment by lllliufeng** [comment]
吃了

**Comment by bangboo12123** [comment]
嗯呢呢嗯呢！（是丰饶令使蜗牛！）
